### PR TITLE
Make API links full paths

### LINKED ISSUE #44: Relative API-doc links are broken
For instace, the "Event Names" link at the bottom of this page doesn't work:

http://guides.emberjs.com/v1.10.0/views/handling-events/

### PR DESCRIPTION
**Pull request by djbender**
Resolves #44

### CODE REVIEW

**Comment by djbender** [comment]
fixed some typos!

File: source/javascripts/vendor/jquery.swiftype.autocomplete.js [hunk 1]
```diff
@@ -41,7 +41,7 @@
       doc_id: docId,
       prefix: value
     };
-    var url = Swiftype.root_url + '/api/v1/public/analytics/pas?' + $.param(params);
+    var url = Swiftype.root_url + 'http://emberjs.com/api/v1/public/analytics/pas?' + $.param(params);
```
Comment (jagthedrummer) [inline source/javascripts/vendor/jquery.swiftype.autocomplete.js]: Not sure about this...

File: source/javascripts/vendor/jquery.swiftype.autocomplete.js [hunk 2]
```diff
@@ -312,7 +312,7 @@
     params['sort_direction'] = handleFunctionParam(config.sortDirection);
     params['per_page'] = config.resultLimit;
 
-    var endpoint = Swiftype.root_url + '/api/v1/public/engines/suggest.json';
+    var endpoint = Swiftype.root_url + 'http://emberjs.com/api/v1/public/engines/suggest.json';
```
Comment (jagthedrummer) [inline source/javascripts/vendor/jquery.swiftype.autocomplete.js]: Or this.

File: source/layout.erb [hunk 3]
```diff
@@ -32,7 +32,7 @@
             <a href="/" class='active'>guides</a>
           </li>
           <li>
-            <a href="http://emberjs.com/api">api</a>
+            <a href="http://http://emberjs.com/api">api</a>
```
Comment (jagthedrummer) [inline source/layout.erb]: Or this.

File: source/models/connecting-to-an-http-server.md [hunk 4]
```diff
@@ -120,7 +120,7 @@ If your JSON API lives somewhere other than on the host root,
 you can set a prefix that will be added to all requests.
 
 For example, if you are using a versioned JSON API, a request for a
-particular person might go to `/api/v1/people/1`.
+particular person might go to `http://emberjs.com/api/v1/people/1`.
```
Comment (jagthedrummer) [inline source/models/connecting-to-an-http-server.md]: This shouldn't be changed, I think.

File: source/models/connecting-to-an-http-server.md [hunk 5]
```diff
@@ -130,7 +130,7 @@ App.ApplicationAdapter = DS.RESTAdapter.extend({
 });
 ```
 
-Requests for a `person` with ID `1`  would now go to `/api/v1/people/1`.
+Requests for a `person` with ID `1`  would now go to `http://emberjs.com/api/v1/people/1`.
```
Comment (jagthedrummer) [inline source/models/connecting-to-an-http-server.md]: Another that shouldn't be changed.

**Comment by djbender** [comment]
@jagthedrummer does yours take care of this? I did a very simple find and replace.

**Comment by jagthedrummer** [comment]
Yeah, mine doesn't contain any of those things.

**Comment by djbender** [comment]
ok i'll withdraw mine :)

**Pull request closed without merging**